### ISSUE TITLE
CSS Problems and Bug

### ISSUE THREAD

**Issue by antwal** [Ju99ernaut/grapesjs-tailwind#12]
Hi, currently to save the css I did this:

```
    editor.Commands.add('save-template', {
      run: function(editor, sender) {
        sender && sender.set('active', 0); // turn off the button
        editor.store();

        var htmldata = editor.getHtml();
        var cssdata = editor.getCss();
        
        var twcss = '';
        editor.runCommand('get-tailwindCss', {
          callback: function(result) {
            twcss = result;
          }
        });

        console.log(htmldata, cssdata, twcss);
      }
    });
```

- Would it be possible that if tailwind components are not used the 'get-tailwindcss' command returns nothing?
- Would it be possible to include the css, directly here instead of retrieving it every time?

<img width="1797" alt="Schermata 2023-11-20 alle 10 00 49" src="https://github.com/Ju99ernaut/grapesjs-tailwind/assets/410736/c51fc3cd-32e4-4615-af7d-84c9f0165e7f">

because I have problems importing the css

- BUG: if instead I create an element, then delete it and recover the css, the previous css is shown

I get the css without any tailwind block:
<img width="2329" alt="Schermata 2023-11-20 alle 10 18 44" src="https://github.com/Ju99ernaut/grapesjs-tailwind/assets/410736/34d65817-9021-49f0-aee3-e71beefae4f1">

I add a tailwind block and get the css:
<img width="2321" alt="Schermata 2023-11-20 alle 10 19 07" src="https://github.com/Ju99ernaut/grapesjs-tailwind/assets/410736/e0737e3d-ec30-43fe-acfe-f58cd16977ba">

I delete all the contents of the editor and get the css again:
<img width="2285" alt="Schermata 2023-11-20 alle 10 19 28" src="https://github.com/Ju99ernaut/grapesjs-tailwind/assets/410736/f967620d-b454-4ce6-a94a-8f007844e68a">

once you have inserted a tailwind block if you no longer want to use it it continues to regerate the css of the last used block

Thank you

**Comment by ronaldohoch** [comment]
Hi, have you discovered a way to do that?

**Comment by antwal** [comment]
> Hi, have you discovered a way to do that?

no, I couldn't solve it

**Comment by Ju99ernaut** [comment]
I don't think we can achieve this completely, as the playCDN is not really meant for production, I have released some utils to help with cleaning up the css.
